ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/SeniorManagerAccess.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018-2023 adorsys GmbH and Co. KG
+ * All rights are reserved.
+ */
+
+package de.adorsys.ledgers.baam.db.domain;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Entity
+@Data
+public class SeniorManagerAccess {
+
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        private String id;
+
+        private String name;
+
+        @Enumerated(EnumType.STRING)
+        private AccessStatus status;
+
+        @ElementCollection
+        private Map<String, String> managerRoles = new HashMap<>();
+
+        // Constructor
+        public SeniorManagerAccess(String name, AccessStatus status) {
+            this.name = name;
+            this.status = status.ACTIVE;
+        }
```

Comment:
You're trying to access the ACTIVE value in the wrong way.

In your constructor, you're passing status as a parameter, and then you use status.ACTIVE. This confuses Java, because it thinks you're trying to get ACTIVE from the status parameter, which doesn't make sense.

**The Fix:**
Instead of using status.ACTIVE, you should directly use the enum name AccessStatus.ACTIVE to get the ACTIVE value.